tests/aws/services/lambda_/test_lambda_performance.validation.json

```diff
@@ -0,0 +1,14 @@
+{
+  "tests/aws/services/lambda_/test_lambda_performance.py::test_number_of_function_versions_async": {
+    "last_validated_date": "2024-02-08T17:18:44+00:00"
+  },
```

Comment:
these tests are a case where having the test duration on AWS might be cool insight as well

Reply:
generally yes; the test duration here against AWS would be rather misleading though: 

```python
num_function_versions = 2 if is_aws_cloud() else 100
```

I think it is important to validate the functionality against AWS, but using a reduced workload saves us lots of 💸  if we ever regularly validate performance tests.